ucx.spec.in

```diff
@@ -13,6 +13,7 @@
 %bcond_with    rocm
 %bcond_with    ugni
 %bcond_with    xpmem
+%bcond_with    java
```

Comment:
this means the default would be to build without JAVA
it should be `%bcond_without` instead - pls fix in UCX master